AutoCollection/Statsbeat.ts

```diff
@@ -81,6 +81,7 @@ class Statsbeat {
                 this._handle.unref(); // Allow the app to terminate even while this loop is going on
             }
             if (!this._longHandle) {
+                this.trackLongIntervalStatsbeats(); // On first enablement
```

Comment:
Should add the catch (error) to avoid the unhandled promise rejection (potential) issue. like below